### ISSUE TITLE
Rio Field Report

### ISSUE THREAD

**Issue by LauraReynal** [MozillaFoundation/plan#258]
Create and distribute field report from Brazil research trip. :earth_americas: 

#### Milestone
https://github.com/MozillaFoundation/research/milestones/Brazil%20Report

#### RACI
Phase: Research / Build / Ship
Owner: @laurareynal, @secretrobotron
Decision: @davidascher
Lead design: n/a
Lead dev: n/a
Quality: @benrito 
With: @benrito @edrushka @HPaulJohnson

**Comment by thisandagain** [comment]
@LauraReynal @benrito do we have a hard deliverable date for this report?

**Comment by LauraReynal** [comment]
Monday 23rd for a first draft @thisandagain

**Comment by benrito** [comment]
I think we’re just trying to wrap it ASAP so the LAN house leaders can get rolling (and we can move forward on other things). They’re going to use it to inform roadmapping and the next phase of work.

It’s a different standard and objective than the other reports so I’m sure we can wrap it soon.

Laura, what are your current thoughts?

Are there anythings I can do to help?

**Comment by thisandagain** [comment]
I won't be available to help with design. Do we need a design resource for this?

**Comment by LauraReynal** [comment]
No we are good for design !

*Laura de Reynal*
Webmaker - Field Research
@lau_nk <https://twitter.com/lau_nk>
Skype: laura-dereynal

**Comment by ldecoursy** [comment]
Hi @LauraReynal Please see here for request for the final file for l10n https://github.com/MozillaFoundation/Mozilla-Learning/issues/95 

It was filed in the Mozilla-Learning repo, however I thought it might be better in webmaker.org repo. Let me know if there's a better place to track that. Thanks!

**Comment by thisandagain** [comment]
Hey @ldecoursy we are now tracking smaller research tasks here: https://github.com/mozillafoundation/research/issues

**Comment by thisandagain** [comment]
@LauraReynal can you clarify what (if anything) is still needed here?

**Comment by LauraReynal** [comment]
Thanks for the Ping Andrew ! 

Im waiting for @benrito 's comments. Do you still want to suggest edits Ben ?
And @thornet ?

Feel free to send me your comments, I will integrate them in the report, and then we can finalize it with a design pass.

**Comment by benrito** [comment]
yeah, will provide edits. might need another day.

**Comment by LauraReynal** [comment]
cc @secretrobotron

**Comment by secretrobotron** [comment]
I don't think it's useful to specify what "LAN" means in this context, since the paragraph is defining what Lan Houses are, not LANs:
> The Lan Houses (‘Local Area Network’) are cyber spaces...

@LauraReynal ?

**Comment by LauraReynal** [comment]
ok for me @secretrobotron


*Laura de Reynal*
Webmaker - Field Research
@lau_nk <https://twitter.com/lau_nk>
Skype: laura-dereynal

**Comment by OpenMatt** [comment]
@LauraReynal status? [[flagged!]](http://plzkthx.herokuapp.com)

**Comment by LauraReynal** [comment]
Waiting for @secretrobotron to finish a copy review before we can publish
Bobby can provide an update

**Comment by secretrobotron** [comment]
My bad. Been preoccupied with Login.

**Comment by OpenMatt** [comment]
* @laurareynal @secretrobotron : status?

**Comment by secretrobotron** [comment]
Login has now shipped. I will proof-read this :). Talked to @benrito a bit about a deadline too: there isn't one, but it's always nice to have earlier than later. I'll make it a priority this week.

**Comment by OpenMatt** [comment]
will work with @HPaulJohnson on blog post

**Comment by secretrobotron** [comment]
##### Page 5
* "allow people to play video games, or to browse the Internet" sounds a little limiting. Where'd this definition come from? Can we add more, like "allow people to do things like play video games, browse the Internet, or even file taxes." ?

##### Page 10
* What is a "Lan House non-user"?

##### Page 11
* for Santa Cruz, "It gained a lot of structure": what is "structure"? Do you mean, "infrastructure"?

**Comment by secretrobotron** [comment]
##### Page 28
* What is "mgh" wrt Internet speed?

**Comment by LauraReynal** [comment]
Thanks for starting the copy review ! :koala: :shipit: 

Page 5 
- Yes to your suggestion, I made the change in the inDesign file. 

Page 10 
Good catch thanks 
- Someone we interviewed to get their perspective on the lan houses, but who doest not go inside the lan house or use their internet. Proposed : Local residents, made the change in the doc 

Page 11 
Yes I think you made the correction already, because I see "infrastructure". Perfect !thansk

**Comment by secretrobotron** [comment]
> What is "mgh" wrt Internet speed?

@soapdog do you know

**Comment by OpenMatt** [comment]
@laurareynal status? [[flagged!]](http://plzkthx.herokuapp.com)

**Comment by secretrobotron** [comment]
Still me. Significant progress. Almost done.

**Comment by soapdog** [comment]
@secretrobotron I think that "mgh" is a typo for "megabytes" as in "20 mb connection".

**Comment by secretrobotron** [comment]
:+1: thanks @soapdog. Makes more sense.

**Comment by thisandagain** [comment]
@secretrobotron Status?

**Comment by secretrobotron** [comment]
Copy edits done. Fixed lots of minor language errors and some arrangement problems. @LauraReynal wanna take a glance now? Should be good to go.

**Comment by LauraReynal** [comment]
@secretrobotron is the latest file 20150304 ?

**Comment by LauraReynal** [comment]
Huge thanks for the copy review @secretrobotron :panda_face: 
I made a few last edits in the file called 20150304. 
If this is the right file then we are good to publish. :heavy_check_mark: 

Would you mind publishing at www.mzl.la/lanhouses
?

**Comment by secretrobotron** [comment]
Looks that way: ![screenshot 2015-05-25 12 29 33](https://cloud.githubusercontent.com/assets/403231/7799842/b9bf236e-02d9-11e5-8a81-5e1bd5cfae37.png)

**Comment by secretrobotron** [comment]
@LauraReynal np. Can you make the PDF?

**Comment by thisandagain** [comment]
Excellent. Thanks folks. Can we cross-publish to http://mzl.la/brazil as well?

**Comment by secretrobotron** [comment]
Not /rio?

@thisandagain I don't have working credentials for mzl.la anymore. Do you?

**Comment by LauraReynal** [comment]
PDF created called : Lan Houses 2015-25-05 :santa: 
Old Files are in Archive 

@secretrobotron @thisandagain   to you now :cactus:

**Comment by secretrobotron** [comment]
Up on S3: https://webmaker-dist.s3.amazonaws.com/reports/rio.pdf

**Comment by thisandagain** [comment]
Published at http://mzl.la/lanhouses
